### PR TITLE
extend use cases

### PR DESCRIPTION
**Pull request by nsheff**
We got some feedback that the use case in the spec were technical, and not really user-oriented.

Also, see #11 

I've done 2 things to address this:

1. I added in a paragraph that describes the primary use cases.
2. I went back through the old user stories document and added in a list of more use-oriented, less technical use cases

I think these provide some useful examples to help give readers an idea of the scope and intent of the standard.

### CODE REVIEW

**Review by tcezard — APPROVED** [review]

File: docs/specification.md [hunk 1]
```diff
@@ -33,16 +33,31 @@ In brief, the project specifies several procedures:
...
+Another weakness of centralized unique identifiers is that they are insufficient to *confirm* identity, which must also consider the content of the genome.
+A related problem is determining compatibility among reference genomes.
+Analytical results based on different genome references may still be integrable, as long as certain conditions about those references are met.
+However, there are no existing tools or standards to formalize and simplify answering the question of reference genome compatibility.
+
+An earlier standard, the refget sequences protocol, partially addressed this issue for individual sequences, such as a single chromosome, but is not directly applicable to collections of sequences, such as a linear reference genome.
+Building on refget sequences, sequence collections presents fundamental concepts, and therefore the specification can be used for many downstream use cases.
+For example, we envision that seqcol identifiers could replace or live alongside the human-readable identifiers currently used to identify reference genomes (e.g. "hg38" or "GRCh38"), which would provide improved reproducibility.
```
Comment (sveinugu) [inline docs/specification.md:49]: Feels redundant with line 36

File: docs/specification.md [hunk 2]
```diff
@@ -33,16 +33,31 @@ In brief, the project specifies several procedures:
...
+This would provide improved reproducibility.
```
Comment (sveinugu) [inline docs/specification.md:50]: Duplicate of end of line 49

File: docs/specification.md (hunk 1 above)
Comment (sveinugu) [inline docs/specification.md:49]: Feels redundant with line 36

File: docs/specification.md [hunk 3]
```diff
@@ -33,16 +33,31 @@ In brief, the project specifies several procedures:
...
 
+Some other examples of common use cases where the use of seqcol is beneficial include:
 
+- As a user I wish to know what sequences are inside a specific collection, so that I can further access those sequences
+- As a user, I want to compare the two sequence collections used by two separate analyses so I can understand how comparable and compatible their resulting data are.
+- As a user I am interested in a genome sequence collection but want to extract those sequences which compose the chromosomes/karyotype of a genome
+- As a submission system, I want to know what exactly a sequence collection contained so I can validate a data file submission.
```
Comment (sveinugu) [inline docs/specification.md:57]:
```suggestion
- As a submission system, I want to know what exactly a sequence collection contains so I can validate a data file submission.
```

File: docs/specification.md [hunk 4]
```diff
@@ -33,16 +33,31 @@ In brief, the project specifies several procedures:
...
+- As a software developer, I want to embed a sequence collection digest in my tool's output so that downstream tools can identify the exact sequence collection that was used
+- As a data processor, my input data didn't include information about the reference genome used, and I want to generate it and attach it so that further processing can benefit from the sequence collection features.
+- I have a chromosome sizes file (a set of lengths and names), and I want to ask whether a given sequence collection is length-compatible with and/or name-compatible with this chromosome sizes file.
```
Comment (sveinugu) [inline docs/specification.md:60]: Should we add the genome browser use case to this list, e.g. I have a sequence collection defining an instance of a genome browser, is this BED compatible?

**Review by sveinugu — CHANGES_REQUESTED** [review]
A few duplications/grammatical errors that should be fixed. Added a suggestion to add a genome browser use case, but that can be ignored if you think it doesn't fit.

Otherwise, LGTM

File: docs/specification.md (hunk 3 above)
Comment (nsheff) [reply docs/specification.md:57]: added

File: docs/specification.md (hunk 4 above)
Comment (nsheff) [reply docs/specification.md:60]: added

**Pull request merged**